src/components/SkyBrowser/SkybrowserTabs.jsx

```diff
@@ -0,0 +1,474 @@
+    showButtonInfoNew[i] = true;
+    this.setState({ showButtonInfoNew });
+    this.setState({ showButtonInfo: showButtonInfoNew });
+  }
+
+  hideTooltip(i) {
+    const showButtonInfoNew = [...this.state.showButtonInfo];
+    showButtonInfoNew[i] = false;
+    this.setState({ showButtonInfoNew });
+    this.setState({ showButtonInfo: showButtonInfoNew });
+  }
+
+  toggleFaceCamera() {
+    const {
+      api, isFacingCamera, selectedBrowser, selectedTarget,
+    } = this.props;
+    const uriBrowser = `ScreenSpace.${selectedBrowser}.FaceCamera`;
+    const uriTarget = `ScreenSpace.${selectedTarget}.FaceCamera`;
+
+    api.setPropertyValueSingle(uriBrowser, !isFacingCamera);
+    api.setPropertyValueSingle(uriTarget, !isFacingCamera);
+  }
+
+  toggleRadiusAzimuthElevation() {
+    const {
+      api, isUsingRae, selectedBrowser, selectedTarget,
+    } = this.props;
+    const uriBrowser = `ScreenSpace.${selectedBrowser}.UseRadiusAzimuthElevation`;
+    const uriTarget = `ScreenSpace.${selectedTarget}.UseRadiusAzimuthElevation`;
+
+    api.setPropertyValueSingle(uriBrowser, !isUsingRae);
+    api.setPropertyValueSingle(uriTarget, !isUsingRae);
+  }
+
+  toggleShowSettings() {
+    this.setState({ showSettings: !this.state.showSettings });
+  }
+
+  setOpacityOfImage(identifier, opacity) {
+    const { skybrowserApi, selectedBrowser } = this.props;
+    skybrowserApi.setOpacityOfImageLayer(selectedBrowser, Number(identifier), opacity);
+  }
+
+  removeImageSelection(identifier) {
+    const { skybrowserApi, selectedBrowser } = this.props;
+    skybrowserApi.removeSelectedImageInBrowser(selectedBrowser, Number(identifier));
+  }
+
+  createButtons(target) {
+    const { skybrowserApi } = this.props;
+    const targetIsLocked = target.isLocked;
+    const targetId = target.id;
+    const toggleSettings = this.toggleShowSettings;
+
+    const lookButton = {
+      selected: false,
+      icon: 'visibility',
+      text: 'Look at target',
+      function(targetId) {
+        skybrowserApi.lockTarget(targetId);
+        skybrowserApi.adjustCamera(targetId);
+      },
+    };
+    const moveButton = {
+      selected: false,
+      icon: 'filter_center_focus',
+      text: 'Move target to center of view',
+      function(targetId) {
+        skybrowserApi.centerTargetOnScreen(targetId);
+      },
+    };
+    const lockButton = {
+      selected: targetIsLocked,
+      icon: 'lock',
+      text: 'Lock aim of target',
+      function(targetId) {
+        if (targetIsLocked) {
+          skybrowserApi.unlockTarget(targetId);
+        } else {
+          skybrowserApi.lockTarget(targetId);
+        }
+      },
+    };
+    const showSettingsButton = {
+      selected: this.state.showSettings,
+      icon: 'settings',
+      text: 'Settings',
+      function(targetId) {
+        toggleSettings();
+      },
+    };
+
+    const buttons = [lookButton, moveButton, lockButton, showSettingsButton];
+
+    const Buttons = buttons.map((button, index) => (
```

Comment:
Not sure it's so great to name this variable with a capital B (implies a class or global constant). Maybe `buttonComponents`?